section1/relection.md

```diff
@@ -1,19 +1,23 @@
 ## Section 1 Reflection
 
 1. How did the SuperLearner Article resonate with you? What from this list do you already do? Want to start doing or do more of? Is there anything not on this list, that you would add to it?
-
+The article surprised me with the idea of taking care of your brain. I already read a lot and take breaks often, but I will now focus on foods that promote brain health. I think that trying to have fun should be added to the list. This helps make learning less of a grind and more of a journey.
```

Comment:
Wow, I love that you are reflecting on how important it is to have fun and to prioritize that alongside studying and working hard.